web-app/src/app/tabs/mppt/mppt.component.html

```diff
@@ -1,3 +1,138 @@
-<p>
-  mppt works!
-</p>
+<div class="mppt">
+	<div class="container-row">
+		<div class="title-left container-column">
+			<span class="subheader">Unit 0</span>
+			<div class="container-row">
+				<div class="container-column title-left text">
+					<div class="subheader">
+						<div class="text">Channel 0</div>
+						<div class="text">Alive</div> 
+						<div>
+			    			<span *ngIf="mppt0.alive; else mppt0AliveElse" class="buttonCentering">
+			    				<img class="circleItems" src="../../../assets/GreenCircle.png" alt="Green Circle" id="circle">
+			    			</span>
+			    			<ng-template #mppt0AliveElse>
```

Comment:
You only need 1 else block for the alive file, that you can place at the beginning. Check the battery page for an example